marta/api.py

```diff
@@ -57,13 +59,14 @@ def get_trains(line=None, station=None, destination=None, api_key=None):
     trains = [t for t in trains if
               (t.line.lower() == line.lower() if line is not None else True) and
               (t.station.lower() == station.lower() if station is not None else True) and
-              (t.destination.lower() == destination.lower() if destination is not None else True)]
+              (t.destination.lower() == destination.lower() if destination is not None else True) and
+              (t.direction.lower() == direction.lower() if direction is not None else True)]
 
     return trains
 
 
 @require_api_key
-def get_buses(route=None, api_key=None):
+def get_buses(route=None, stop_id: int = None, bus_id: int = None, timepoint: str=None, api_key=None):
```

Comment:
This looks good, but I have a couple of comments:
- this could use a docstring similar to `get_trains` that describes the parameters
- The types for stop and vehicle ID should technically be strings (in MARTA's feed they all parse to integers pretty nicely, but I think they're essentially string identifiers)
- I'd like to see `bus_id` change to `vehicle` just so that the argument names continue the trend of matching what's in the API response.

Reply:
Docstring added, bus_id changed to vehicle_id. I'm going to push back on the stop and vehicle ID being strings; it's important to limit them to ints if they can only be numbers.